iris-mpc-gpu/tests/e2e.rs

```diff
@@ -360,15 +363,54 @@ mod e2e_test {
                                 right: db.db[deleted_idx as usize].clone(),
                             }
                         }
-                        // TODO: implement test that enables testing the bitmap OR rule setting.
-                        // This will require setting the left and right iris code to be different,
-                        // one of them below the threshold and the other
-                        // above the threshold against the other codes. It will
-                        // also be required to set the param BatchQuery.threshold_bitmap_or to true
-                        // for the specific iris codes to be matched against
-                        // 5 => {
-                        // println!("Sending iris codes that match on right but not left with the OR
-                        // rule set");
+                        5 => {
+                            println!(
+                                "Sending iris codes that match on right but not left with the OR \
```

Comment:
This println no longer fits with the logic below

Reply:
i'll improve the wording to match the implementation, thanks1